### PR TITLE
Disambiguate some xrefs

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Part of [sdk](https://github.com/dwavesystems/dwave-ocean-sdk/pull/300), [dimod](https://github.com/dwavesystems/dimod/pull/1372), [dwave-cloud-client](https://github.com/dwavesystems/dwave-cloud-client/pull/633), and [dwave-system](https://github.com/dwavesystems/dwave-system/pull/524) PRs to reduce build warnings

### CODE REVIEW

File: docs/intro/index.rst [hunk 1]
```diff
@@ -4,32 +4,33 @@
...
+    :hidden:
 
-   overview
-   using
-   developing
-   reference_examples
+    overview
+    using
+    developing
+    reference_examples
```
Comment (randomir) [inline docs/intro/index.rst:36]: Frivolous indentation changes are best coupled with bigger semantic changes to not pollute/add noise to git history. Alternatively, one can separate formatting changes in a separate commit.

**Review by randomir — APPROVED** [review]

File: docs/intro/index.rst (hunk 1 above)
Comment (JoelPasvolsky) [reply docs/intro/index.rst:36]: I regret not fixing the indentation in a separate commit. I deny the charge of friviolity.
